### ISSUE TITLE
Support on Flutter for using accessibilityIdentifier (UIAccessibilityIdentification) on iOS?

### ISSUE THREAD

**Issue by asalmasi** [flutter/flutter#21140]
Do the Semantics class in Flutter provide anyway to map Widgets to an accessibilityIdentifier on iOS side? We rely on Appium for testing and the Flutter driver is not sufficient for meeting our testing needs. Is there any plan on the road-map for Flutter to support this soon?

**Comment by jonahwilliams** [comment]
@asalmasi I would love it if you would share what features flutter_driver needs or any improvements we could make. Regarding `UIAccessibilityIdentification`, we don't have a way to provide a stable mapping to that value from widgets and no plans for that to change.

**Comment by jonahwilliams** [comment]
Sorry for the delay in getting back to you, to summarize:

We do produce UIAccessibilityElements from a widget tree and flutter should be as accessible as a native app (modulo some bugs we have filled and if you find any please file issues). From my understanding, UIAccessibilityIdentification is not required for accessibility, it is just used for automation scripts.  Perhaps we need a better strategy for incorporating flutter tests into native tests? 

Is it not possible to test your BLE interaction using a platform_channel and flutter_driver?

cc @matthew-carroll

**Comment by Wanghuayin** [comment]
@jonahwilliams do you have any suggestion for this issue? [Does the flutter driver hava plans to support the hybrid app?](https://github.com/flutter/flutter/issues/22809)

**Comment by zoechi** [comment]
Is this a dup of #17988 ?
